### ISSUE TITLE
Please Fix The Download Issue

### ISSUE THREAD

**Issue by wadeceol32** [Anistream/Anistream-Mobile#158]
You were my only anime salvation, so please fix this as soon as possible.

**Comment by ghostlygamer7** [comment]
Yeah I've been trying to use the app but either it won't download or it says something is blocking your vpn

**Comment by XDgamerCZ** [comment]
Developer given up. It's dead forever and no one will ever gona fix it! Switch to Crunchyroll or 9Anime if i can suggest.
